internal/mode/static/state/dataplane/configuration.go

```diff
@@ -601,12 +607,14 @@ func generateCertBundleID(configMap types.NamespacedName) CertBundleID {
 
 // buildTelemetry generates the Otel configuration.
 func buildTelemetry(g *graph.Graph) Telemetry {
-	if g.NginxProxy == nil || g.NginxProxy.Spec.Telemetry == nil || g.NginxProxy.Spec.Telemetry.Exporter == nil {
+	if g.NginxProxy == nil ||
```

Comment:
shall we check is g.NginxProxy.Valid is true before we use NginxProxy in this file?